src/lib/y2network/routing.rb

```diff
@@ -38,5 +38,16 @@ def initialize(tables:)
     def routes
       tables.flat_map(&:to_a)
     end
+
+    # Determines whether two set of routing settings are equal
+    #
+    # @param other [Routing] Routing settings to compare with
+    # @return [Boolean]
+    def ==(other)
+      forward_ipv4 == other.forward_ipv4 && forward_ipv6 == other.forward_ipv6 &&
+        (tables - other.tables).empty?
```

Comment:
same here. I really do not like assymetric `==` which can result that `a == b` but `b != a`

Reply:
```
irb(main):001:0> a = [1,2]
=> [1, 2]
irb(main):002:0> b = [2]
=> [2]
irb(main):003:0> (a-b).empty?
=> false
irb(main):004:0> (b-a).empty?
=> true
```